### ISSUE TITLE
Change definition and add parts to NOTE for "target offset"

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2427]
Target offset now reads:
https://w3c.github.io/wcag/guidelines/22/#dfn-target-offsets

(Also change in Understanding! https://w3c.github.io/wcag/understanding/target-size-minimum.html#dfn-target-offset)

> the distance measured from the farthest point of a target to the nearest point of the second target. Offset includes the target and spacing around the target. The target offset from A to B may be different then the offset from B to A, if the size of these targets differ.

Now as the definition text can be replaced in a SC by the definition itself it should be changed to:

> distance measured from the farthest point of a target to the nearest point of the second target.
> 
> NOTE
> Offset includes the target and spacing around the target.
> 
> NOTE
> The target offset from A to B may be different than the offset from B to A, if the size of these targets differ.

If changed the SC reads as (changed definition to text of definition):

> Spacing: The distance measured from the farthest point of a target to the nearest point of the second target is at least 24 CSS pixels to every adjacent target;

**Comment by alastc** [comment]
I think the convention is that the first sentence is the one that would replace the word. However, having the other parts as part of the definition makes them normative.

I think the second sentence as normative makes sense, but the last part could be a note.
